### PR TITLE
Clear entry if pressing down arrow with last msg in entry

### PR DESCRIPTION
**Pull request by ThePhilgrim**
https://user-images.githubusercontent.com/76887896/147389843-2222fba7-0bdf-4e13-a19d-54e0bdc3baaa.mov

### CODE REVIEW

**Comment by ThePhilgrim** [comment]
Doesn't exactly fix #103 . However, I believe it is already fixed, and this is related to it.

File: tests/test_messaging.py [hunk 1]
```diff
@@ -119,13 +119,17 @@ def test_history(alice, bob, wait_until):
     assert not alice.entry.get()
     alice.previous_message_to_entry()
     assert alice.entry.get() == "first message"
+    alice.next_message_to_entry()
+    assert not alice.entry.get()
+
+    assert not alice.entry.get()
```
Comment (Akuli) [inline tests/test_messaging.py:125]: There's no need to assert it twice. Delete one of the two asserts.

File: tests/test_messaging.py [hunk 2]
```diff
@@ -119,13 +119,17 @@ def test_history(alice, bob, wait_until):
...
     alice.previous_message_to_entry()
     assert alice.entry.get() == "first message"
-    alice.next_message_to_entry()
+    alice.previous_message_to_entry()
     assert alice.entry.get() == "first message"
```
Comment (Akuli) [inline tests/test_messaging.py:129]: Why does the message stay there when you press down arrow key? This doesn't seem correct to me.

Comment (ThePhilgrim) [reply tests/test_messaging.py:129]: It is staying when you press _up_ arrow, not down.

I'm just trying to keep the test logic from master branch:

```

    assert not alice.entry.get()
    alice.previous_message_to_entry()
    assert alice.entry.get() == "first message"
    alice.previous_message_to_entry()
    assert alice.entry.get() == "first message"
    alice.next_message_to_entry()
    assert alice.entry.get() == "first message"
    alice.next_message_to_entry()
```

Here, you press up-arrow to display "first message", then you press up-arrow again, and the content in entry remains the same. Then you press down-arrow. With the logic before, "first message" would still be there. This is what I removed on line 124-127, and I'm keeping the logic of pressing up-arrow twice before pressing down-arrow. However, as you see on the new line 131, it's asserting that it's empty.

**Review by Akuli — APPROVED** [review]

**Pull request merged**